docs/staking-modules/csm/guides/addresses.md

```diff
@@ -98,4 +100,33 @@ These changes grant `managerAddress` the ultimate control over the Node Operator
 
 :::info
 If the `extendedManagerPermissions` option is used, `managerAddress` should be set to a secure address ("cold" wallet or multisig)
-:::
\ No newline at end of file
+:::
+
+## Examples
+
+### Default
+
+By default, the Node Operator's creator is set as a `managerAddress` and `rewardAddress`.
+
+![addresses-2](../../../../static/img/csm/addresses-2.png)
+
+It is recommended that addresses be re-assigned according to one of the optimal configurations below.
```

Comment:
Following the creation of the node operator (i.e. once at least one key has been registered), it is recommended that...